src/parser/parser.ts

```diff
@@ -84,6 +84,32 @@ const getEventName = (eventAttributeString: RMLEventAttributeName): [RMLEventNam
  * };
  * ```
  **/
+function stringifyInitialValue(initialValue: any, attributeName?: string): string {
```

Comment:
`initialValue` shouldn't be of type `any`, but the types we actually expect to come through.